test/assert.js

```diff
@@ -44,28 +47,37 @@ function locateAssertion (error) {
  * @see https://github.com/vowsjs/vows#assertistrueactual-message
  * @see https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html
  */
-function assert (actual, expected, ...messages) {
-  // in case console.assert gets converted without changing == to ,: fail
-  // it makes no sense to call this method with only one boolean argument
-  if (typeof actual === 'boolean' && arguments.length === 1) throw new Error('convert equal to ,')
-  let caught
-  // the assertion with the better error message comes first so we can benefit
-  try {
-    node_assert.strict.equal(actual, expected)
-  } catch (error) {
-    if (!isAssertionError(error)) throw error
-    caught = error
-  }
-  // the original assertion comes second, it has to always be executed
-  node_assert(actual == expected, ['(loose ==)', ...messages, actual, locateAssertion(caught)].join(' '))
-  if (caught) {
-    const msg = ['(strict ===) ', ...messages, caught.message, locateAssertion(caught)]
-    if ((this.strictThrows || assert.strictThrows)()) {
-      caught.message = msg.join(' ')
-      throw caught
-    }
-    (this.log || assert.log)(...msg)
-  }
+function assert(actual, expected, ...messages) {
+	// in case console.assert gets converted without changing == to ,: fail
+	// it makes no sense to call this method with only one boolean argument
+	if (typeof actual === 'boolean' && arguments.length === 1)
+		throw new Error('convert equal to ,')
+	let caught
+	// the assertion with the better error message comes first so we can benefit
+	try {
+		node_assert.strict.equal(actual, expected)
+	} catch (error) {
+		if (!isAssertionError(error)) throw error
+		caught = error
+	}
+	// the original assertion comes second, it has to always be executed
+	node_assert(
+		actual == expected,
+		['(loose ==)', ...messages, actual, locateAssertion(caught)].join(' ')
+	)
+	if (caught) {
+		const msg = [
+			'(strict ===) ',
+			...messages,
+			caught.message,
+			locateAssertion(caught),
+		]
+		if ((this.strictThrows || assert.strictThrows)()) {
+			caught.message = msg.join(' ')
+			throw caught
+		}
+		;(this.log || assert.log)(...msg)
```

Comment:
I don't know why this semicolon is here. But since we are getting rid of this whole file soon ...

Reply:
FYI the rationale is given here: https://prettier.io/docs/en/rationale.html#semicolons